src/common/components/cards/how-to-fix-android-card-row.tsx

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { Term } from 'assessments/markup';
+import { FormattedResolution } from 'common/types/store-data/unified-data-interface';
+import { isEmpty, isNil } from 'lodash';
+import { CardRowProps } from '../../../common/configs/unified-result-property-configurations';
+import { NamedFC } from '../../../common/react/named-fc';
+import { SimpleCardRow } from './simple-card-row';
+
+export interface HowToFixAndroidCardRowProps extends CardRowProps {
```

Comment:
**not for this PR but**: by inheriting from `CardRowProps`, `HowToFixAndroidCardRowProps` contains as a deps a:
```typescript
    fixInstructionProcessor: FixInstructionProcessor;
```
which means, we should be able to add the colored square for color contrast failures.
Again, I don't think we need to do that as part of this PR

Reply:
i will be sending another pr for thi, once this is merged

Reply:
i have added support for color processing now.